.kitchen.dokken.yml

```diff
@@ -1,17 +1,14 @@
 driver:
   name: dokken
   privileged: true # because Docker and SystemD/Upstart
-  chef_version: current
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
 
 transport:
   name: dokken
 
 provisioner:
   name: dokken
 
-verifier:
```

Comment:
It's duplication of code, the `.kitchen.dokken.yml` overwrites stuff in `.kitchen.yml`, so no need to define things twice.

In an ideal world the former just containers the driver, transport and platforms, rest is defined in the latter.